docs/src/app/components/pages/components/paper.jsx

```diff
@@ -64,7 +64,7 @@ const PaperPage = React.createClass ({
           },
           {
             name: 'zDepth',
-            type: 'number (0-5)',
+            type: 'oneOf ["0", "1", "2", "3", "4", "5"]',
```

Comment:
https://github.com/callemall/material-ui/blob/master/src/utils/prop-types.js#L26

I have noticed that you add `"` where it's number. Why ?